### ISSUE TITLE
The bottom settings menu is not visible. 

### ISSUE THREAD

**Issue by Zeyadas** [koreader/koreader#10236]
* KOReader version: 3.2023
* Device: Samsung galaxy F13, android version 13

#### Issue: The bottom settings menu is not visible.  
![Screenshot_20230322_104720](https://user-images.githubusercontent.com/70109540/226835826-acbd7be2-d13c-44a5-92bb-bc9e41ad8e45.jpg)

I don't know if it is a duplicate and solved earlier, I'm a survival of the northen Syria earthquake.

**Comment by poire-z** [comment]
Duplicate of many, but no problem :) good you are ok (ok enough to get upgraded to Android 13 I assume :))
See https://github.com/koreader/koreader/issues/9446#issuecomment-1297542495

**Comment by Zeyadas** [comment]
Getting fast respond from you as usual. upgraded to android 13 with a broken leg and shoulder.
And yeah I lost everything but my family and I still alive.
I had another chance to use Koreader, thank you guys.

**Comment by Zeyadas** [comment]
Didn't work for me, can you please make it easier for me? As go to settings > something....

**Comment by Frenzie** [comment]
Other than seeing which effect `Developer options > Display cutout > Render apps below cutout area` has I don't think there's any kind of solution available yet unfortunately.

**Comment by hius07** [comment]
Look at this patch
https://github.com/koreader/koreader/issues/9446#issuecomment-1238296520

**Comment by foundObjects** [comment]
Just wanted to add that the OnePlus 9 Pro is also affected. I don't normally read on my phone but it's nice to have available if I'm stuck somewhere waiting on something.

The user patch above does make the footer settings menu fully visible until this issue is officially fixed.

**Comment by pazos** [comment]
Closing this as a dupe of #9446

Also happy to see you around, @Zeyadas
